kiva/fonttools/_scanner.py

```diff
@@ -0,0 +1,538 @@
+# (C) Copyright 2005-2021 Enthought, Inc., Austin, TX
```

Comment:
correct me if im wrong but it looks like this module has two distinct functions - `FontEntry` and the code to create `FontEntry` instances from font files and the code to scan system fonts.

Is it overkill to split these two functions into separate modules?

Reply:
I'll look into this, but I won't do it in this PR.